addons/base_address_extended/models/res_country.py

```diff
@@ -4,15 +4,10 @@
 from odoo import fields, models
 
 
-class ResCountry(models.Model):
+class Country(models.Model):
```

Comment:
useless diff

Reply:
it's just for consistency and uniformity